lib/matplotlib/axes/_base.py

```diff
@@ -2428,6 +2428,7 @@ def autoscale_view(self, tight=None, scalex=True, scaley=True):
         case, use :meth:`matplotlib.axes.Axes.relim` prior to calling
         autoscale_view.
         """
+        print('asv', scalex, scaley)
```

Comment:
@timhoffm I missed these prints :(
Can you revert them?  Thanks.

Reply:
Woops, will do tonight.